src/__integrationtests__/Ctypes.spec.ts

```diff
@@ -2,49 +2,61 @@
  * @group integration/ctype
  */
 
-import { faucet } from './utils'
+import { wannabeFaucet } from './utils'
 import CType from '../ctype/CType'
 import ICType from '../types/CType'
 import { getOwner } from '../ctype/CType.chain'
 import getCached from '../blockchainApiConnection'
 import { Identity } from '..'
 
-describe('When there is an CtypeCreator and a verifier', async () => {
-  const CtypeCreator = faucet
+describe('When there is an CtypeCreator and a verifier', () => {
+  let ctypeCreator: Identity
+  let ctypeCounter = 0
 
-  const ctype = CType.fromCType({
-    schema: {
-      $id: 'http://example.com/ctype-10',
-      $schema: 'http://kilt-protocol.org/draft-01/ctype#',
-      properties: {
-        name: { type: 'string' },
-      },
-      type: 'object',
-    } as ICType['schema'],
-  } as ICType)
+  function makeCType(): CType {
```

Comment:
it's better that each test case uses their own ctype. This reduces dependencies between test cases.